### PR TITLE
fix, confusing info/error message when exporting bookmarks

### PR DESCRIPTION
**Pull request by mustafa-001**
When multiple bookmarks exported Koreader shows following message/error:
nothingwasexportedbug

This PR removes `all_count == 1` check (holds `false` with multiple books). 

I guess it was added to decide which message to display but string template already does that.

---
This change is [Reviewable](https://reviewable.io/reviews/koreader/koreader/5765)

### CODE REVIEW

File: plugins/evernote.koplugin/main.lua [hunk 1]
```diff
@@ -575,25 +575,21 @@ function EvernoteExporter:exportClippings(clippings)
...
-                N_("Exported notes from the book:\n%1",
-                   "Exported notes from the book:\n%1\nand %2 others.",
-                   all_count-1),
-                export_title,
-                all_count-1
-            )
-        end
+        msg = T(
+            N_("Exported notes from the book:\n%1",
+                "Exported notes from the book:\n%1\nand %2 others.",
```
Comment (Frenzie) [inline plugins/evernote.koplugin/main.lua]: Would you mind keeping the indentation as it was for aesthetic reasons? (Not that this isn't logical.)

Comment (mustafa-001) [reply plugins/evernote.koplugin/main.lua]: Don't know why but both VS Code and Vim gave me a fight over one character.

File: plugins/evernote.koplugin/main.lua [hunk 2]
```diff
@@ -575,25 +575,21 @@ function EvernoteExporter:exportClippings(clippings)
...
-                   "Exported notes from the book:\n%1\nand %2 others.",
-                   all_count-1),
-                export_title,
-                all_count-1
-            )
-        end
+        msg = T(
+            N_("Exported notes from the book:\n%1",
+               "Exported notes from the book:\n%1\nand %2 others.",
+                all_count-1),
```
Comment (Frenzie) [inline plugins/evernote.koplugin/main.lua]:
```suggestion
               all_count-1),
```

File: plugins/evernote.koplugin/main.lua [hunk 3]
```diff
@@ -575,25 +575,21 @@ function EvernoteExporter:exportClippings(clippings)
...
-                   "Multiple errors occurred while trying to export notes from the book:\n%1\nand %2 others.",
-                   error_count-1),
-                error_title,
-                error_count-1
-            )
-        end
+        msg = T(
+            N_("An error occurred while trying to export notes from the book:\n%1",
+               "Multiple errors occurred while trying to export notes from the book:\n%1\nand %2 others.",
+                error_count-1),
```
Comment (Frenzie) [inline plugins/evernote.koplugin/main.lua]:
```suggestion
               error_count-1),
```

**Comment by Frenzie** [comment]
Or I would but you I guess you didn't tick the write permission box.

**Pull request merged**